sidekiq-rate-limiter.gemspec

```diff
@@ -28,4 +28,5 @@ Gem::Specification.new do |s|
 
   s.add_dependency "sidekiq", ">= 4.0", "< 6.0"
   s.add_dependency "redis_rate_limiter"
+  s.add_dependency "redis-semaphore"
```

Comment:
gem authors are more likely to accept a PR that doesn't add a dependency, especially when there are numerous redlock-like implementations